Makefile

```diff
@@ -17,7 +17,7 @@ install-hooks:
 .PHONY: init-hooks
 
 ## build: Build celestia-node binary.
-build:
+build: fetch-tags
```

Comment:
can you remove fetch-tags from every other target?

Reply:
should the build only contain it?

Then it is no sense to have target